en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md

```diff
@@ -204,15 +204,16 @@ A couple things to note here:
 
 * By assigning a value of `true` to the field, we defer to the model to infer the type for the field. To override that, we can always add a `type` property:
 
-```yaml
-MyProject\Models\Product:
-  fields:
-    onSale:
-      type: Boolean
-```
+    ```yaml
+    MyProject\Models\Product:
+    fields:
+        onSale:
+        type: Boolean
```

Comment:
Is it correct? `onSale` property doesn't have any value?

Reply:
Oops, I missed an extra level of indentation there. Will fix it.

Reply:
Done